src/tests/dictionaryConcepts/container/SelectAnswers.test.js

```diff
@@ -13,22 +17,59 @@ describe('<SelectAnswers />', () => {
       source: 'test source',
       frontEndUniqueKey: 'unique',
     };
-    wrapper = shallow(<SelectAnswers {...props} />);
+    wrapper = mount(<SelectAnswers {...props} />);
   });
 
-  it('should handle change in AsncSelect', () => {
-    const instance = wrapper.instance();
-    const spy = jest.spyOn(instance, 'handleChange');
-    instance.handleChange(['value']);
+  it('should handle key down event when a user presses enter button to search for concept', async () => {
+    const spy = jest.spyOn(wrapper.instance(), 'handleKeyPress');
+    const event = { keyCode: KEY_CODE_FOR_ENTER };
+    await wrapper.instance().handleKeyPress(event, 'malaria');
     expect(spy).toHaveBeenCalled();
-    instance.handleChange(null);
+  });
+
+  it('should handle key down event when a user presses enter button to search for concept with input less that 3', async () => {
+    const spy = jest.spyOn(wrapper.instance(), 'handleKeyPress');
+    const event = { keyCode: KEY_CODE_FOR_ENTER };
+    await wrapper.instance().handleKeyPress(event, 'ma');
     expect(spy).toHaveBeenCalled();
   });
 
-  it('should update the state with current input value', () => {
-    wrapper = mount(<SelectAnswers {...props} />);
-    const instance = wrapper.instance();
-    instance.handleInputChange('searchTerm');
-    expect(wrapper.state().inputValue).toEqual('searchTerm');
+  it('should handle key down event when a user presses button thats not enter to search for concept', () => {
+    const spy = jest.spyOn(wrapper.instance(), 'handleKeyPress');
+    const event = { keyCode: 9 };
```

Comment:
What key does code 9 represent? Can you use a constant for it too?

Reply:
I have updated it to use the key code for space which is a constant named `KEY_CODE_FOR_SPACE` in the helperFunction.js